openedx/adg/common/lib/zoom_client/client.py

```diff
@@ -95,3 +98,49 @@ def delete_webinar(self, webinar):
         response = self.zoom_client.webinar.delete(**webinar.get_zoom_formatted_data())
         log.info(f'Deleting webinar, title: {webinar.title} response: {response.status_code}')
         return response.status_code == HTTP_204_NO_CONTENT
+
+    def _get_account_information(self):
+        """
+        Get the account_number and account_id information
+
+        Returns:
+            dict: A dict containing the account information
+        """
+        response = self.zoom_client.user.get(id='me')
+        if response.status_code == HTTP_404_NOT_FOUND:
+            return
+
+        account_data = json.loads(response.content)
+        account_id = account_data.get('account_id')
+        account_number = account_data.get('account_number')
+
+        return {'account_id': account_id, 'account_number': account_number}
+
+    def check_alternative_host_permissions(self, alternative_hosts):
+        """
+        Check if the alternative hosts are licensed users and belong to the same account as
+        the host
+
+        Args:
+            alternative_hosts (str): A comma separated list of emails belonging to alternative hosts
+
+        Returns:
+            bool: True if all the emails belong to valid users with required permissions to be
+            an alternative host else False
+        """
+        account_information = self._get_account_information()
+        if not account_information:
+            return False
+
+        alternative_hosts_emails = extract_emails_from_string(alternative_hosts)
+        for email in alternative_hosts_emails:
+            response = self.zoom_client.user.get(id=email)
+            alternative_host_data = json.loads(response.content)
+            if (
+                response.status_code == HTTP_404_NOT_FOUND
+                or alternative_host_data.get('type') != self.LICENSED_ACCOUNT_TYPE
+                or alternative_host_data.get('account_id') != account_information.get('account_id')
+                or alternative_host_data.get('account_number') != account_information.get('account_number')
+            ):
+                return False
```

Comment:
I think we can move this to a new helper method in this zoomclient, That might be useful in future as the task being performed here can be used somewhere. WDYT?

Reply:
I agree, creating a new method for it.